pom.xml

```diff
@@ -136,6 +136,7 @@
         <swagger.version>2.1.11</swagger.version>
         <swagger.scala.module.version>2.5.2</swagger.scala.module.version>
         <swagger-ui.version>4.1.0</swagger-ui.version>
+        <trino.client.version>363</trino.client.version>
```

Comment:
Why choose this version? Does trino has a client/server compatible matrix？

Reply:
It is a relatively recent version of trino-client, the latest version is 365. Each version will have a client, but there are few changes. There seems to be no compatible matrix.